tokenserver-common/src/error.rs

```diff
@@ -271,6 +271,14 @@ impl ReportableError for TokenserverError {
                 TokenType::BrowserId => Some("request.error.browser_id".to_owned()),
                 TokenType::Oauth => Some("request.error.oauth".to_owned()),
             }
+        } else if matches!(
+            self,
+            TokenserverError {
+                status: "invalid-client-state",
+                ..
+            }
+        ) {
+            Some("request.error.invalid_client_state".to_owned())
```

Comment:
It's probably worth having a different metric (or preferably tag, but we don't have `ReportableError::tags` in this codebase yet which let's you do that) for the specific "Unacceptable client-state value stale value" issue in question.

Though I suppose we can alternatively look at the warn log messages for a count of those 🤷‍♂️

Reply:
Yeah, let's hold off on adding a tag for that. (Huh, I thought I had a PR that converted all the errors to impl `InternalError`, but maybe I'm confusing that with the other repo.

In any case, I'll add a different ticket to do that and we can add the tag once that lands.